### PR TITLE
run leave script as super user

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- [run leave script as super user](https://github.com/airqo-platform/AirQo-api/pull/1327/commits/1df00867baefb9b5c96d581bc2f4cdb52366dc9b)
- Update data-mgt api node preference to compute node

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- n/a

**_HOW DO I TEST OUT THIS PR?_**
- n/a

### CODE REVIEW

File: infra/ansible/playbooks/k8s/preemptive-worker.yml [hunk 1]
```diff
@@ -50,4 +50,4 @@
       cron:
         name: "Reset k8s node"
         minute: "9"
-        job: "/bin/bash /root/leave_k8s_cluster.sh"
+        job: "sudo /bin/bash /root/leave_k8s_cluster.sh"
```
Comment (123MwanjeMike) [inline infra/ansible/playbooks/k8s/preemptive-worker.yml:53]: [run leave script as super user](https://github.com/airqo-platform/AirQo-api/pull/1327/commits/1df00867baefb9b5c96d581bc2f4cdb52366dc9b)

File: k8s/data-mgt/templates/deployment.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ spec:
                   - key: node-type
                     operator: In
                     values:
-                      - general-purpose
+                      - compute
```
Comment (123MwanjeMike) [inline k8s/data-mgt/templates/deployment.yaml:38]: Update data-mgt api node preference to compute node

File: infra/ansible/playbooks/k8s/preemptive-worker.yml (hunk 1 above)
Comment (Baalmart) [reply infra/ansible/playbooks/k8s/preemptive-worker.yml:53]: Thanks @123MwanjeMike , much appreciated!
What challenge is this super user privilege addressing?

Comment (123MwanjeMike) [reply infra/ansible/playbooks/k8s/preemptive-worker.yml:53]: Hi @Baalmart 
The challenge was that the leave script was running the `kubeadm reset` command which can only be run with administrative rights.

**Review by NoahNsimbe — COMMENTED** [review]
LGTM

**Review by NoahNsimbe — APPROVED** [review]

**Pull request merged**